doc/getting-started.rst

```diff
@@ -539,7 +537,7 @@ With ``update-versions-file``, whenever Buildout gets the newest
 version for a requirement (subject to requirement constraints), it
 appends the version to the named file, along with a comment saying
 when and why the requirement is installed.  If you later want to
-upgrade a dependency, just edit this file with the new version, or to
+upgrade a dependency, manually update this file with the new version. Alternatively,
```

Comment:
"just edit" is fine. "Manually update" suggests that your can also do it automatically.